gstchecksumsink.c

```diff
@@ -142,6 +146,11 @@ gst_cksum_image_sink_class_init (GstCksumImageSinkClass * klass)
           "save decoded raw frames into file (YV12 and I420 only)",
           FALSE, G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
 
+  g_object_class_install_property (gobject_class, PROP_RAW_LOCATION,
+      g_param_spec_string ("dump-location", "File Location",
+          "Location of the file to write decoded raw frames",
+          NULL, G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
+
```

Comment:
Hi Artie,

What about G_PARAM_CONSTRUCT_ONLY since it cannot be re-assigned while running.

Reply:
It needs to be set from open_raw_file (via g_file_open_tmp if not user specified).  If I add this param spec, then I get warning:
```
GLib-GObject-WARNING **: g_object_set_is_valid_property: construct property "dump-location" for object 'GstCksumImageSink' can't be set after construction
```

Reply:
I always forget that in gstreamer ... sorry